pkgs/development/libraries/raylib/default.nix

```diff
@@ -56,18 +56,18 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.tests = [ raylib-games ];
 
   patches = [
-    # Patch version in CMakeList to 4.5.0
+    # Patch version in CMakeList to 5.0.0
     # Remove this when updating to a new revision
     (fetchpatch {
-      url = "https://github.com/raysan5/raylib/commit/0d4db7ad7f6fd442ed165ebf8ab8b3f4033b04e7.patch";
-      hash = "sha256-RGokbQAwJAZm2FU2VNwraE3xko8E+RLLFjUfDRXeKhA=";
+      url = "https://github.com/raysan5/raylib/commit/032cc497ca5aaca862dc926a93c2a45ed8017737.patch";
+      hash = "sha256-qsX5AwyQaGoRsbdszOO7tUF9dR+AkEFi4ebNkBVHNEY=";
     })
   ];
 
   # fix libasound.so/libpulse.so not being found
   preFixup = ''
-    ${lib.optionalString alsaSupport "patchelf --add-needed ${alsa-lib}/lib/libasound.so $out/lib/libraylib.so.${finalAttrs.version}"}
-    ${lib.optionalString pulseSupport "patchelf --add-needed ${libpulseaudio}/lib/libpulse.so $out/lib/libraylib.so.${finalAttrs.version}"}
+    ${lib.optionalString alsaSupport "patchelf --add-needed ${alsa-lib}/lib/libasound.so $out/lib/libraylib.so"}
+    ${lib.optionalString pulseSupport "patchelf --add-needed ${libpulseaudio}/lib/libpulse.so $out/lib/libraylib.so"}
```

Comment:
Why is this needed?

Reply:
I don't know if it's still needed. I don't for which reason it's was added in the first place.

I removed the `finalAttrs.version` because the git tag did not match with the build version `5,0,0 vs 5,0` so i was not sure i should hard code or just remove it

Reply:
Ok i just checked and libraries are not even linked anymore

```
> ldd result/lib/libraylib.so.4.5.0 | grep libpulse                                                                          
	/nix/store/ldyiwysirfdxww141v649lslpq9xm0c1-libpulseaudio-16.1/lib/libpulse.so (0x00007f78fa33d000)
	libpulsecommon-16.1.so => /nix/store/ldyiwysirfdxww141v649lslpq9xm0c1-libpulseaudio-16.1/lib/pulseaudio/libpulsecommon-16.1.so (0x00007f78f9b6e000)
> ldd result/lib/libraylib.so.4.5.0 | grep libasound                                                                         
	/nix/store/3ivh62ailk57r674fhk6nx1bg2nhirr7-alsa-lib-1.2.11/lib/libasound.so (0x00007f510608e000)
	
> ldd result/lib/libraylib.so.5.0.0 | grep libasound
> ldd result/lib/libraylib.so.5.0.0 | grep libpulse
```